README.md

```diff
@@ -155,6 +57,12 @@ Use "picotool help <cmd>" for more info
 
 Note commands that aren't acting on files require a device in BOOTSEL mode to be connected.
 
+## Building & Installing
+
+If you don't want to build picotool yourself, you can find pre-built executables for Windows, macOS, and Linux in the [pico-sdk-tools](https://github.com/raspberrypi/pico-sdk-tools/releases) repository. Assuming you've unzipped that directory to `$PICOTOOL_LOC` (with the actual picotool executable at `$PICOTOOL_LOC/picotool/picotool`), you can point the Pico SDK at this binary by setting the `picotool_DIR` environment variable to `$PICOTOOL_LOC/picotool`, or by passing `-Dpicotool_DIR=$PICOTOOL_LOC/picotool` to your `cmake` command or setting it in your `CMakeLists.txt` file.
```

Comment:
Maybe this would be slightly less confusing if you used `$SDKTOOLS_LOC` instead of `$PICOTOOL_LOC` ? (assuming of course that I haven't misunderstood what's going on! :wink: )

Reply:
Perhaps `$EXTRACT_LOCATION` ? My concern with `$PICOTOOL_LOC/picotool/picotool` is that people might find it confusing with the word "picotool" appearing 3 times?